.github/CODEOWNERS

```diff
@@ -1461,6 +1461,7 @@ x-pack/test/api_integration/apis/management/index_management/inference_endpoints
 /x-pack/test/plugin_functional/test_suites/resolver/ @elastic/security-solution
 /x-pack/test/security_solution_api_integration @elastic/security-solution
 /x-pack/test/api_integration/apis/security_solution @elastic/security-solution
+/x-pack/test/functional/es_archives/entity/host_risk @elastic/security-solution @elastic/security-detection-engine
```

Comment:
I wonder, why this archive has `@elastic/security-solution` in ownership, but `/x-pack/test/functional/es_archives/entity/risks` only detection-engine?

Reply:
### es_archives/entity/host_risk

So for `x-pack/test/functional/es_archives/entity/host_risk`, it's only used once, according to `CMD+SHIFT+F`:

<img width="1728" alt="Screenshot 2024-10-03 at 12 22 48 (2)" src="https://github.com/user-attachments/assets/c65bd390-d2bf-42b4-8608-3af8bb6dc1c0">

### es_archives/entity/risks

While most || all of the usages of `x-pack/test/functional/es_archives/entity/risks` are under `x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine`
<img width="1718" alt="Screenshot 2024-10-03 at 12 25 47" src="https://github.com/user-attachments/assets/fc5a338f-502c-45f8-b17c-e337f6ef452c">

_Honestly, I'm making poorly educated guess_

That said, my team and I are eager to get 💯 percent ownership, for improved reporting.  
If you've any ideas @vitaliidm , we'd love the help!  😅

Reply:
What caught my eye, both archives used under the same path `detections_response/detection_engine/**`, but they have different ownerships. That's why I asked question.
I would say, let's put both of them under only `@elastic/security-detection-engine`, since this group is a part of 
`@elastic/security-solution` anyway.

Would be also great, to move `/x-pack/test/functional/es_archives/entity/host_risk` under `/x-pack/test/functional/es_archives/entity/risks`, in the same section